build/yamls/flow-aggregator/base/flow-aggregator.yml

```diff
@@ -5,6 +5,75 @@ metadata:
   name: flow-aggregator
 ---
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flow-aggregator-ca
+  namespace: flow-aggregator
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: flow-aggregator
+  namespace: flow-aggregator
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: flow-aggregator-role
+  namespace: flow-aggregator
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    resourceNames: ["flow-aggregator-ca"]
+    verbs: ["get", "update"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "create", "update"]
```

Comment:
I understand we cannot specify resourceName for create verb, but we should still limit the get and update permission to `flow-aggregator-client-tls` only. They can be written in this way:
```
  - apiGroups: [""]
    resources: ["secrets"]
    resourceNames: ["flow-aggregator-client-tls"]
    verbs: ["get", "update"]
  - apiGroups: [""]
    resources: ["secrets"]
    verbs: ["create"]
```

Reply:
Updated. Thanks!